testing/retool-test.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -euf -o pipefail
+
+# Change working directory
+cd /ms-playwright
+
+# Generate the tests
+node ./generate.js
```

Comment:
ooc - do we have to generate these files because playwright doesn't have js api like cypress? (only cli that expects test files to exist?)

Reply:
Playwright has a robust [JavaScript API](https://playwright.dev/docs/intro). If we used that, we'd lose access to all the features of the Playwright testing framework, which includes test reporting, parallel test workers, and setup and tear down. I think it's worth the extra preprocessing work to generate the tests files and use the existing test runner.

Reply:
Got it - thanks for the context 👍  I agree, we can see how much more complex (or not) this generation step gets over time to evaluate that tradeoff again later